src/hooks/use-music-api.js

```diff
@@ -0,0 +1,21 @@
+import { useCallback } from "react";
+import { useEnv } from "../store/env-context";
+import useGetRequest from "./use-get-request";
+
+const useMusicApi = () => {
+  const { musicApi } = useEnv();
+  const {
+    data: tracks,
+    isLoading,
+    error,
+    sendRequest,
+  } = useGetRequest(musicApi);
+
+  const getTracks = useCallback(async (name) => {
+    await sendRequest(`/Tracks?name=${name}`);
+  }, [sendRequest]);
+
```

Comment:
I thought it would be a good idea to encapsulate all requests to the Music API inside of the useMusicApi hook. So, here I would define other functions like `getTrackById`, `createTrack`, and so on. Maybe it would be better to create a **separate** hook for each of them (`useGetTracks`, useGetTrackById`, `useCreateTrack`)? Or there is no need to create those wrapping hooks at all?